### DISCUSSION TITLE
Use multiple cameras with this repo

### DISCUSSION THREAD

**Discussion by DominikSch93** [AxisCommunications/acap-integration-examples-aws#20] [General]
Hi,

I want to use this repository for multiple cameras but still with only one CloudFormation stack. Is there a way to distinguish between different cameras that are sending requests to the same API Gateway address?
Does anyone has experience with it?

Happy about any answer :)

Regards
Dominik

**Comment by mattias-kindborg-at-work** [comment]
Hi @DominikSch93!

Are we talking about the example sending images to S3? If yes, then the same AWS infrastructure can be shared among many cameras. You'd only need to provision the AWS resources once, while performing the camera configuration once per camera.
